server/controller/src/main/java/ai/starwhale/mlops/common/RegExps.java

```diff
@@ -26,18 +24,4 @@ public class RegExps {
 
     public static final String BUNDLE_NAME_REGEX = "^[a-zA-Z][a-zA-Z\\d_-]{2,80}$";
```

Comment:
How about moving `RegExps.BUNDLE_NAME_REGEX` to somewhere that is closer to it's usage such as `BundleAccessor`.

Reply:
Regexes have been moved to service classes